frontend/assets/stylesheets/components/_action.scss

```diff
@@ -97,6 +97,29 @@
     cursor: default;
 }
 
+.action--snap {
```

Comment:
Not really related to the spacing vars, but I might take another look at this to get rid of the magic numbers.